.azure-pipelines/js-mgmt-ci.yml

```diff
@@ -53,3 +53,27 @@ jobs:
       - task: PublishBuildArtifacts@1
         inputs:
           pathtoPublish: $(Build.SourcesDirectory)/drop
+
+  - job: 'Analyze'
+
+    pool:
+      vmImage: 'Ubuntu 16.04'
+
+    steps:
+      - task: NodeTool@0
+        inputs:
+          versionSpec: '$(NodeVersion)'
+        displayName: 'Install Node.js $(NodeVersion)'
+
+      - task: Npm@1
+        displayName: 'npm install'
+        inputs:
+          verbose: false
```

Comment:
The default is `verbose: false`, so you should remove all instances of `verbose: false`.

Reply:
done